src/helpers/statusBarHelper.ts

```diff
@@ -34,5 +34,5 @@ export const isUsingButtonNavigation = () => {
   const deviceHeight = Dimensions.get('screen').height;
   const windowHeight = Dimensions.get('window').height;
   const bottomNavBarHeight = deviceHeight - windowHeight;
-  return bottomNavBarHeight > 70;
+  return bottomNavBarHeight > 95;
```

Comment:
any good way to not have this hardcoded? not sure exactly how this is being used

Reply:
not that I'm aware. There's probably a better way to check what navigation mode an Android user is using though..

Reply:
is this helper basically guessing whether the android has a soft nav bar or not? wondering if we can just get the nav bar height and return whether it's nonzero